doc/developer-guides/GVT-g-porting.rst

```diff
@@ -44,47 +44,46 @@ documentation.
 Overall Components
 ******************
 
-For the GVT-g solution for the ACRN  hypervisor, there are two
-key modules: AcrnGT and GVT:
+For the GVT-g solution for the ACRN  hypervisor, there are two key modules:
+AcrnGT and GVT.
 
 AcrnGT module
   Compiled from ``drivers/gpu/drm/i915/gvt/acrn_gvt.c``, the AcrnGT
   module acts as a glue layer between the ACRN hypervisor and the
   interface to the ACRN-DM in user space.
 
-  AcrnGT is the agent of hypervisor related services, including I/O trap
+  AcrnGT is the agent of hypervisor-related services, including I/O trap
   request, IRQ injection, address translation, VM controls, etc. It also
-  listens to ACRN hypervisor in ``acrngt_emulation_thread``, and informs GVT
-  module of I/O traps.
+  listens to ACRN hypervisor in ``acrngt_emulation_thread``, and informs the
+  GVT module of I/O traps.
 
   It calls into the GVT module's :ref:`intel_gvt_ops_interface` to invoke
-  Device Model's routines, and receives request from GVT module through
+  Device Model's routines, and receives requests from the GVT module through
   the :ref:`MPT_interface`.
 
-  User space programs, such as ACRN-DM, communicate with AcrnGT through
+  User-space programs, such as ACRN-DM, communicate with AcrnGT through
   the :ref:`sysfs_interface` by writing to sysfs node
   ``/sys/kernel/gvt/control/create_gvt_instance``.
 
-  This is the only module that needs to be rewritten when porting to
-  another embedded device hypervisor.
+  This is the only module that must be rewritten when porting to another
+  embedded device hypervisor.
 
 GVT module
-  This Device Model service is the central part of all
-  GVT-g components. It receives workloads from each vGPU, shadows the
-  workloads, and dispatches the workloads to Dom0's i915 module to deliver
-  workloads to real hardware. It also emulates the virtual display to each
-  VM.
+  This Device Model service is the central part of all the GVT-g components.
+  It receives workloads from each vGPU, shadows the workloads, and
+  dispatches the workloads to Dom0's i915 module to deliver workloads to
```

Comment:
Dom0 is a Xen term, we should be using "Service VMM' here instead I assume?

Reply:
Changed accordingly.